src/main/resources/application.yaml

```diff
@@ -0,0 +1,16 @@
+spring:
+  datasource:
+    username: ${db_username}
```

Comment:
use parameter expansion to add some default values e.g. `${db_username:postgres}`

Reply:
It seems to be corrected, but I didn't really understood what exactly I needed to do